### DISCUSSION TITLE
Deprecated DOMNodeInserted event listener now logs as an error.

### DISCUSSION THREAD

**Discussion by KirstenAli** [vueup/vue-quill#578] [General]
As of today (at least in my browser, Google Chrome v127.0.6533.72), the deprecated DOMNodeInserted event now logs as an error. It appears that the DOMNodeInserted event has finally been fully removed in Chrome.

**Comment by sminogue** [comment]
Has there been any response to this? My chrome still shows this error 6 months after the question was posted. I'm not sure its worth the time integrating this WYSIWYG editor into my project if it can just crap out at any time?

**Comment by luthfi-hh** [comment]
Thanks for reporting this.

I rechecked this against the current release. The deprecated `DOMNodeInserted` listener came from older installs that resolved Quill 1.x; Quill 1.3.7 registers `DOMNodeInserted` in its scroll blot. Current `@vueup/vue-quill@1.5.5` depends on Quill 2 (`2.0.2 || >=2.0.4 <3`), where the editor uses `MutationObserver` instead.

I also verified the Vite example in npm-package mode on Chrome 149: the editors mounted, typing worked, and there were no console warnings/errors or deprecated mutation-event listeners.

If anyone still sees this, please upgrade `@vueup/vue-quill` and make sure your lockfile resolves `quill@2.x`. Closing this as resolved.
